webapp/src/features/UserItem/BillingHistory.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { Button, Image } from 'react-bootstrap'
+import Cards from '../../widgets/Cards'
+import Table from '../../widgets/Table'
+import Icons from '../../assets/icons'
+import { useDispatch, useSelector } from 'react-redux'
+import { setSelectedUserItemModal } from './userItem.slice'
+import { selectedUserItemModal } from './userItem.selectors'
+import { USERITEM_MODAL_TYPES } from '../UserItemModals/UserItemModals.constants'
+import getUserItemModal from '../UserItemModals'
+
+const BillingHistory = () => {
+	const dispatch = useDispatch()
+	const selectedModal = useSelector(selectedUserItemModal)
+
+	const deleteButtonFormatter = (cell, row) => (
+		<div className='d-inline-flex align-items-center'>
+			{row.revenue === '$115' && (
```

Comment:
please add a TODO here that we need to fix this

Reply:
updated